lib/subject_selector.rb

```diff
@@ -58,7 +58,10 @@ def missing_subject_set_error
   end
 
   def default_page_size
```

Comment:
I think "default_page_size" is a bit of a misnomer given that it can be overridden using params.